src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -82,6 +71,14 @@ void HBProcessor::process(const cpp2::HBReq& req) {
     } else if (nebula::error(lastUpdateTimeRet) == nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
         resp_.set_last_update_time_in_ms(0);
     }
+
+    auto version = metaVersion_.load();
+    if (version == -1) {
+        metaVersion_.store(static_cast<int64_t>(MetaVersionMan::getMetaVersionFromKV(kvstore_)));
```

Comment:
I think you could initialize it when metad startup.

Reply:
Yeah, when metad startup, it has already set this version. Just to avoid use a global.